changelogs/unreleased/gh-5158-invalid-recovery-from-xlog.md

```diff
@@ -0,0 +1,4 @@
+## bugfix/replication
+
+* Fixed replication recovery from the xlogs after rebootstraping replica
```

Comment:
Let's be more specific: `Fixed an assertion failure on master when replica resubscribes with a smaller vclock than previously seen`.

Reply:
Done